yamllint/cli.py

```diff
@@ -201,6 +201,8 @@ def run(argv=None):
 
     for file in find_files_recursively(args.files, conf):
         filepath = file[2:] if file.startswith('./') else file
+        if(conf.is_file_ignored(filepath) or None):
```

Comment:
```suggestion
        if conf.is_file_ignored(filepath):
```
I don't understand how `or None` would help.

Reply:
Perhaps change the commit text to:
```
cli: match symlinks to ignore early on, before following them
```